pkg/server/server.go

```diff
@@ -417,3 +422,37 @@ func (s *Server) handleOpenShiftTokenDeletion(user *auth.User, w http.ResponseWr
 	io.Copy(w, resp.Body)
 	resp.Body.Close()
 }
+
```

Comment:
Is it possible to move your `handlers` into your `helm package`?  A `handlers.go` file would be fine, and just load them up via imports.  

`server.go` doesn't need to know any details about `helm`, it can include the 3 lines setting up the `handle("api/console/helm/....", authHandlerWithUser( helmcatalog.InstallHandler )`.  etc.

Reply:
Would still like to see this change.